frontend/pendingTranslations.ftl

```diff
@@ -191,6 +191,12 @@ phone-banner-resend-welcome-sms-cta = Resend welcome SMS
 phone-banner-resend-welcome-sms-title = Quick Tip
 phone-banner-resend-welcome-sms-body =  Remember to save the contact we shared with you by SMS to help you identify messages forwarded by { -brand-name-relay }. Can’t find it?  
 
+# Phone What's New
+whatsnew-feature-phone-header = Introducing phone number masking
+whatsnew-feature-phone-snippet = With { -brand-name-relay-premium }, you can now create a phone number mask that helps you get...
+whatsnew-feature-phone-description = With { -brand-name-relay-premium }, you can now create a phone number mask that helps you get texts and calls without revealing your true number.
```

Comment:
It's a bit confusing that this says that Premium offers phone masks, even though the regular Premium plan doesn't.

Reply:
Hmm, I got a bit confused too. I thought upgrading to Premium would upgrade you to Phones as well. Thanks for the clarification.

Reply:
Checking with Chris that the strings are correct. Since everything's in `pendingTranslations` it'll be easy to update though.